server/controllers/index.js

```diff
@@ -0,0 +1,7 @@
+// const Recipe = require('./recipes');
```

Comment:
So it's actually not really good practice to push commented code. Please remove it, commit and then push again